test/gen_casadi_test.py

```diff
@@ -3015,6 +3015,56 @@ def test_signed_expression(self):
         casadi_model.simplify(compiler_options)
         self.assertEqual(ineg.value, -1.0)
 
+    def test_resolve_param_expressions(self):
+        """Test that both + and - prefix operators work in expressions"""
```

Comment:
Looks like the comment needs an update after copy/paste. (Not marked draft, so assuming you were ready for review.)

Reply:
Good catch on the bug and nice to see more test coverage! 

Looks good to me other than the above.